src/Simplex/Chat/Store/Connections.hs

```diff
@@ -111,7 +111,7 @@ getConnectionEntity db vr user@User {userId, userContactId} agentConnId = do
           chatSettings = ChatSettings {enableNtfs = fromMaybe MFAll enableNtfs_, sendRcpts, favorite}
           mergedPreferences = contactUserPreferences user userPreferences preferences $ connIncognito conn
           activeConn = Just conn
-       in Right Contact {contactId, localDisplayName, profile, activeConn, viaGroup, contactUsed, contactStatus, chatSettings, userPreferences, mergedPreferences, createdAt, updatedAt, chatTs, contactGroupMemberId, contactGrpInvSent, uiThemes, chatDeleted, customData}
+       in Right Contact {contactId, localDisplayName, profile, activeConn, viaGroup, contactUsed, contactStatus, chatSettings, userPreferences, mergedPreferences, createdAt, updatedAt, chatTs, contactGroupMemberId, contactGrpInvSent, chatTags = [], uiThemes, chatDeleted, customData}
```

Comment:
can't it provide incorrect state to UI? if core sends toView event that triggers chat update.

Reply:
there may be other similar state traps, for example toContact, toGroup - we'd have to exhaustively check that entities built from any usages of those are not passed anywhere in response to be then updated in ui

Reply:
better check other functions too, and only set chatTags to empty list where the entity is new (for example createDirectContact, createAcceptedContact should be ok)